app/services/edition_taxon_publisher.rb

```diff
@@ -0,0 +1,32 @@
+class EditionTaxonPublisher
```

Comment:
Maybe `EditionTaxonLinkPatcher` would more clearly indicate what this class does?

Reply:
I realise that doesn't match the previous method name, but I think that is another good change you'd be making.

Reply:
Sounds good